service_container.rst

```diff
@@ -344,15 +344,18 @@ you can type-hint the new ``SiteUpdateManager`` class and use it::
     // ...
     use App\Service\SiteUpdateManager;
```

Comment:
```suggestion
    use App\Service\SiteUpdateManager;
    // ...
```

Sorry, this is extremely nitpicking-me, but the collapsed use statements now includes at least 1 `Symfony` namespace, which should be located after the `App` namespace (so the `// ...` is placed after the existing use statement).

Reply:
Okey. Thank you